ssc/Pos/Ssc/NistBeacon.hs

```diff
@@ -49,7 +49,7 @@ instance Ssc SscNistBeacon where
     sscCreateNodeContext = Tagged $ const (pure ())
 
 instance SscHelpersClass SscNistBeacon where
-    sscVerifyPayload = const $ const $ Right ()
+    sscVerifyPayload _ _ = Right ()
```

Comment:
yeah, it's one of those that I made for consistency and then was like “maybe I shouldn't”